apps/src/sites/code.org/pages/public/teacher-dashboard/sections.js

```diff
@@ -59,3 +60,33 @@ export function unmountSyncOauthSectionControl() {
 function syncOauthSectionMountPoint() {
   return document.getElementById('react-sync-oauth-section');
 }
+
+/**
+ * Render the login type details and controls for changing login type
+ * at the bottom of the manage students tab.
+ * @param {number} sectionId
+ */
+export function renderLoginTypeControls(sectionId) {
+  registerReducers({teacherSections});
+  const store = getStore();
+
+  store.dispatch(asyncLoadSectionData());
```

Comment:
Would it make sense to check and see if we already have section data, and if so, not async load it again?